dask_kubernetes/operator/daskcluster.py

```diff
@@ -237,9 +238,12 @@ async def daskworkergroup_update(spec, name, namespace, logger, **kwargs):
             )
         logger.info(f"Scaled worker group {name} up to {spec['replicas']} workers.")
     if workers_needed < 0:
+        with rpc(f"tcp://{cluster_name}.{namespace}:8786") as scheduler:
+            scheduler_info = scheduler
+        logger.info(f"Scheduler info: {scheduler_info}")
```

Comment:
Is this the right way to connect to the scheduler with dask rpc? In the logs I get this:
```
[2022-02-24 07:26:24,129] kopf.objects         [INFO    ] [default/default-worker-group] Scheduler info: <rpc to 'tcp://10.244.0.46:8786', 0 comms>
```
We shouldn't expect there to be zero comms, right?

Reply:
That's just because you haven't used the RPC yet so no comms have been opened.